### PR TITLE
Meteor app setup

### PR DESCRIPTION
**Pull request by lattln**
initial app setup for meteor with react.

### CODE REVIEW

File: imports/ui/ThemeToggle.jsx
Comment (jengledow) [inline imports/ui/ThemeToggle.jsx:1]: have you thought about putting the theme into a context so we can pass it to every child? just a thought

**Review by jengledow — APPROVED** [review]
looks like the start to a meteor project to me

File: imports/ui/ThemeToggle.jsx
Comment (lattln) [reply imports/ui/ThemeToggle.jsx:1]: daisyUI theme is already being passed inside the tailwind config file

**Comment by AustinDecker** [comment]
not gud enough

**Review by AustinDecker — APPROVED** [review]

**Review by Nguyen-Noah — APPROVED** [review]
image

**Review by CollinFoord — APPROVED** [review]

**Pull request merged**